spec/models/aggregated_cash_outgoings_spec.rb

```diff
@@ -366,7 +366,7 @@ def error_msg(name, month)
         let(:subject) { aco.update(corrected_valid_params) }
 
```

Comment:
There's a few other references to Dates that haven't been updated to the new standard, i.e. line 50 `Date.new(2020, 12, 1).in_time_zone`? and 384, `Date.today.at_beginning_of_month` should be `Date.current.at_beginning_of_month`